.spelling

```diff
@@ -77,6 +77,7 @@ init
 labeled
  - demos/Azure/README.md
 alpha.11
```

Comment:
The others are included in the docs and aren't really worth updating. I removed alpha.11 since it was not in the docs.